book/_toc.yml

```diff
@@ -24,6 +24,12 @@ parts:
           title: Albedo
           sections:
             - file: tutorials/albedo/aviris-ng-data
+        - file: tutorials/snowex-core/index
+          title: Data Overview
+          sections:
+            - file: tutorials/snowex-core/01_snowex_data_overview
+            - file: tutorials/snowex-core/02_snowex_data_descriptions
+            - file: tutorials/snowex-core/03_snowex_data_preview  
```

Comment:
The real file name is `snex_data_preview`

That is where the current build is failing

Reply:
Looks like the error now is related to MyST?